### PR TITLE
Add uuid generator

### PR DESCRIPTION
**Pull request by VasylHryha**
Close: https://github.com/epicmaxco/vuestic-ui/issues/1021
## Description

Changes:
- feat add UUID generator
- feat add test for UUID generator
- feat remove nodeBuiltinsPlugin

## Markup:

<details>

```vue
// Your code
```
</details>
New UUID looks like it:

image

esm node_modules before:

image

after:

image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/services/utils.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 //  @ts-nocheck
 
-import { v4 as uuidv4 } from 'uuid'
```
Comment (VasylHryha) [inline packages/ui/src/services/utils.ts:3]: `uuid` module isn't specified in a package.json file
only in yarn.lock

File: packages/ui/src/services/utils.ts [hunk 2]
```diff
@@ -98,4 +97,10 @@ export const getProp = <T extends (Record<string, unknown> | string)> (option: T
   return option
 }
 
-export const generateUuid = () => uuidv4()
+const getRandomString = (stringLength = 4): string => {
+  return Math.random().toString(36).substring(2, stringLength + 2)
+}
+
+export const generateUuid = () => {
```
Comment (VasylHryha) [inline packages/ui/src/services/utils.ts:104]: actually it's not uuid because have a different format now, change the naming `generateUuid ` ?

Comment (asvae) [reply packages/ui/src/services/utils.ts:104]: `generateUniqueId`?

File: packages/ui/build/rollup/plugins/rollup-teaser-preset.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { terser, Options } from 'rollup-plugin-terser'
```
Comment (VasylHryha) [inline packages/ui/build/rollup/plugins/rollup-teaser-preset.js:1]: I got warning in console that Options  declared but not used, so I removed it
but now I see that it used to define type `{ Options }`
will revert

Comment (m0ksem) [reply packages/ui/build/rollup/plugins/rollup-teaser-preset.js:1]: It was used on 3d line as JSDoc.

File: packages/ui/build/rollup/plugins/rollup-teaser-preset.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
+import { terser } from 'rollup-plugin-terser'
 
 /** @param { Options } options  */
```
Comment (m0ksem) [inline packages/ui/build/rollup/plugins/rollup-teaser-preset.js:3]:
```suggestion
```

File: packages/ui/build/rollup/plugins/rollup-teaser-preset.js (hunk 3 above)
Comment (m0ksem) [reply packages/ui/build/rollup/plugins/rollup-teaser-preset.js:1]: You can try to use `import type { Options }` here to prevent this warning. (Maybe)

**Review by m0ksem — CHANGES_REQUESTED** [review]

File: packages/ui/build/rollup/plugins/rollup-teaser-preset.js (hunk 4 above)
Comment (m0ksem) [reply packages/ui/build/rollup/plugins/rollup-teaser-preset.js:3]: You can remove it, but better if you try to import type in additional.

**Comment by m0ksem** [comment]
image

Tested build

File: packages/ui/src/services/utils.ts (hunk 1 above)
Comment (m0ksem) [reply packages/ui/src/services/utils.ts:3]: Yap. It is not going to final bundle. 👍

**Review by m0ksem — APPROVED** [review]

**Review by asvae — APPROVED** [review]
Wow, that took out a lot of deps :D.
Thanks.

**Pull request merged**